articles/defender-for-cloud/quickstart-onboard-github.md

```diff
@@ -90,7 +90,7 @@ The Defender for Cloud service automatically discovers the organizations where y
 > [!NOTE]
 > To ensure proper functionality of advanced DevOps posture capabilities in Defender for Cloud, only one instance of a GitHub organization can be onboarded to the Azure Tenant you are creating a connector in.
 
-The **DevOps security** pane shows your onboarded repositories grouped by Organization. The **Recommendations** pane shows all security assessments related to GitHub repositories.
+The **DevOps security** pane shows your onboarded repositories grouped by Organization. The **Recommendations** pane shows all security assessments related to GitHub repositories.  The onboarding may take up to 8 hours to show the initial DevOps security results.
```

Comment:
```suggestion
Upon successful onboarding, DevOps resources (e.g., repositories, builds) will be present within the Inventory and DevOps security pages. It may take up to 8 hours for resources to appear. Security scanning recommendations may require [an additional step to configure your pipelines](azure-devops-extension.md). Refresh intervals for security findings vary by recommendation and details can be found on the Recommendations page.
```